examples/getting_started/rag_custom_data.py

```diff
@@ -0,0 +1,27 @@
+from typing import List
+
+from haystack.document_stores import InMemoryDocumentStore
+from haystack.pipeline_utils import build_rag_pipeline, build_indexing_pipeline
+from haystack.pipeline_utils.indexing import download_files
+
+document_store = InMemoryDocumentStore()
+indexing_pipeline = build_indexing_pipeline(
+    document_store=document_store,
+    embedding_model="sentence-transformers/all-mpnet-base-v2",
+    supported_mime_types=["text/plain", "text/html"],
+)
+
+# before indexing, let's get some documents from the web
+files: List[str] = download_files(sources=["http://www.paulgraham.com/superlinear.html"])
```

Comment:
can we remove the type annotation and also the import above? 

Reason: we should keep this getting started code as minimal and short as possible.